magento/magento2
# Fix incorrect empty cart button behavior

### Description

PR changes order of cart form submit elements to have Update Shopping Cart first in the DOM, to make sure it is used as submit button when Enter key is pressed. There is also change to the theme styling to adjust Clear Shopping Cart position after changing buttons order.

### Fixed Issues

1. magento/magento2#21499

### Manual testing scenarios

1. Add any product to cart
2. Navigate to the cart
3. Un-hide "Clear Shopping Cart" button which is hidden in Luma by default by running this in browser console:
  `jQuery("#empty_cart_button").show();`
4. Change item quantity in the input
5. Press "Enter"

Expected result is to have product quantity updated.

### Contribution checklist
